src/base/index.vue

```diff
@@ -573,21 +690,41 @@ export default {
       // We check if this is running in the browser or not
       // because during SSR no cors preflight request is sent
       const inBrowser = typeof window !== 'undefined';
+
+      const queryStringObject = {}
+      if(Object.keys(this.$route.query).length !== 0) {
+        Object.keys(this.$route.query).map((queryKey) => {
+          queryStringObject[queryKey] = this.$route.query[queryKey]
+        })
+      }
+
       const config = {
         headers: sectionHeader(((inBrowser) ? {} : {origin: this.$sections.projectUrl})),
       };
       const URL =
         this.$sections.serverUrl +
         `/project/${this.$sections.projectId}/page/${this.pageName}`;
 
+      // TODO: Send Query string payload only when it is requested to be used (Use Sections ENV for it)
```

Comment:
It should not be based on the ENV but on the content of the page itself

Any reason why this can't be part of this release ?